JoinColony/colonyCDapp
# Fix: Update Mobile Sidebar Route Accordion Item behaviour

## Description

![mobile-accordion](https://github.com/user-attachments/assets/399201ba-b634-401b-9d87-a0b977bc7ebf)

## Testing

1. Click Finances
2. Verify that
 - the Sidebar is not dismissed
 - the Accordion items are revealed
3. Click Balances
4. Verify that
 - the Sidebar is dismissed
 - you are taken to the Balances page
 
Resolves #3098
